tests/base/discrete_time_1.cc

```diff
@@ -69,7 +69,7 @@ test_adjust_time_step_size()
   print_time(time);
   time.advance_time();
   print_time(time);
-  time.set_desired_next_step_size(0.36);
```

Comment:
Note that like 77 still has `set_desired_next_step_size()` so we do not lose test coverage.